scripts/upload-stat.py

```diff
@@ -0,0 +1,65 @@
+#! /usr/bin/env python3
+#
+# Parse the stat.txt file from lang-stat and upload to the metrics collection endpoint
```

Comment:
Thank you!

(we could have made the shell script produce json, instead of parsing the `txt` file here)

Reply:
yeah I thought about it but figured it wasn't that hard to parse your format and I like your human readable format